src/overlays/effects/ovl_Effect_Ss_Stone1/z_eff_ss_stone1.c

```diff
@@ -6,35 +6,77 @@
 
 #include "z_eff_ss_stone1.h"
 
-typedef enum {
-    /* 0x00 */ SS_STONE1_0,
-    /* 0x01 */ SS_STONE1_1,
-    /* 0x02 */ SS_STONE1_2,
-    /* 0x03 */ SS_STONE1_3,
-    /* 0x04 */ SS_STONE1_4,
-    /* 0x05 */ SS_STONE1_5,
-    /* 0x06 */ SS_STONE1_6,
-    /* 0x07 */ SS_STONE1_7,
-    /* 0x08 */ SS_STONE1_8,
-    /* 0x09 */ SS_STONE1_9,
-    /* 0x0A */ SS_STONE1_A,
-    /* 0x0B */ SS_STONE1_B,
-    /* 0x0C */ SS_STONE1_C,
-} EffectSsStone1Regs;
+#define rReg0 regs[0]
 
 u32 EffectSsStone1_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
 void EffectSsStone1_Draw(GlobalContext* globalCtx, u32 index, EffectSs* this);
 void EffectSsStone1_Update(GlobalContext* globalCtx, u32 index, EffectSs* this);
 
-/*
 EffectSsInit Effect_Ss_Stone1_InitVars = {
     EFFECT_SS_STONE1,
     EffectSsStone1_Init,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/effects/ovl_Effect_Ss_Stone1/EffectSsStone1_Init.s")
+typedef struct {
+    /* 0x00 */ void* texture;
+    /* 0x04 */ Color_RGBA8 primColor;
+    /* 0x08 */ Color_RGBA8 envColor;
+} EffStoneDrawInfo;
+
+static EffStoneDrawInfo sDrawInfo[] = {
+    { 0x04029A90, { 200, 0, 0, 255 }, { 0, 0, 0, 255 } },
```

Comment:
Why did you decide to not add a symbol for this gameplay keep data?

Reply:
in oot at least, we only add stuff to undefined syms if its needed for proper code generation. It will get a proper symbol/name when objects are handled proper.
I know this is different in mm tho. Roz and I had a conversation about this pretty recently in mm-decomp-help